doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
+
+- **Network Service (NS):** composition of **Network Function**(s) and/or **Network Service**(s), defined by its functional and behavioural specification, including the service lifecycle.
+
+- **Virtual Application (VA):** A general term for software which can be loaded into a Virtual Machine.
+  >_*Note:*_ a **VNF** is one type of VA.
+
+- **Virtual Machine (VM):** virtualised computation environment that behaves like a physical computer/server.
+  >_*Note:*_ A **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server. It is created using Instance Type together with sizing information or Compute Flavour.  
+
+- **Virtual Machine instances Catalogue:** Pre-defined instance types and compute flavours.  
+
+- **Virtual Network Function (VNF):** a software implementation of a **Network Function**, capable of running on the **Cloud Infrastructure**.
+  - **VNF**s are built from one or more VNF Components (**VNFC**) and, in most cases,  the VNFC is hosted on a single VM or Container.
+
+
+- **Virtual resources:**
+  - **Virtual Compute resource (a.k.a. virtualisation container):** partition of a compute node that provides an isolated virtualised computation environment.
+  - **Virtual Storage resource:** virtualised non-volatile storage allocated to a virtualised computation environment hosting a **VNFC**
+  - **Virtual Networking resource:** routes information among the network interfaces of a virtual compute resource and physical network interfaces, providing the necessary connectivity
+
+
+- **Workload:** Workload refers to software running on top of compute resources such as **VMs** or **Containers**. Most relevant workload categories in context of cloud infrastructure are:
+  - **Data Plane Workloads:** are related to packet handling of the end-to-end communication between applications. These tasks are expected to be very I/O and memory read/write operations intensive.
+  - **Control Plane Workloads:** are the task related to any other communication between NFs that is not directly related to the end-to-end data communication between applications. For example, this category includes session management, routing or authentication.
+  - **Storage Workloads:** are all tasks related to disk storage (either SSD or HDD or other).  Examples range from non-intensive router logging to more intensive database read/write operations.
 
 <a name="1.2"></a>
 ## Hardware Layer Terminology
 
-- **Physical Network Function (PNF)**: Implementation of a network function via tightly coupled dedicated hardware and software system.
+- **Cloud Infrastructure Hardware Profile:** defines the behaviour, capabilities and metrics provided by an cloud infrastructure Hardware Layer.
```

Comment:
For alphabetical, please move after "Cloud Infrastructure Hardware Configuration"

Reply:
thanks, done